backend/models.py

```diff
@@ -13,11 +13,17 @@ class User(db.Model):
 
     __tablename__ = "user"
     id = Column(Integer, autoincrement=True, primary_key=True)
-    username = Column(String(200), unique=True)
+    github = Column(String(200), unique=True)
     password = Column(String(100))
+    first_name = Column(String(50))
+    last_name = Column(String(50))
+    email = Column(String(200))
+    phone = Column(String(13))
+    slack = Column(String(21))
+    is_admin = Column(Boolean)
```

Comment:
Please take a look at #386 and #388 to consider if these matters should be solved in this PR or later on.